api/v1/ptpconfig_webhook.go

```diff
@@ -118,6 +122,12 @@ func (r *PtpConfig) validate() error {
 		if summary_interval != logSyncInterval {
 			return errors.New("summary_interval " + summary_interval + " must match logSyncInterval " + logSyncInterval)
 		}
+
+		if profile.PtpSchedulingPolicy != nil && *profile.PtpSchedulingPolicy == "SCHED_FIFO" {
```

Comment:
do you need to check also about `SCHED_OTHER`?

Reply:
My opinion is that we don't.  If SCHED_OTHER, no validation is necessary.